spec/lib/tacos_demo.py

```diff
@@ -213,25 +213,15 @@ def parse_comments(
 def parse_comment(
```

Comment:
can you write a docstring with an example of the transformation that is happening here, it's seems quite complex and is not easy to read

Reply:
a unit test is a good match for that kind of situation, too